index.js

```diff
@@ -111,10 +111,6 @@ File.prototype.clone = function(opt) {
   // Clone our file contents
   var contents;
   if (this.isStream()) {
-    if (typeof this.contents.clone !== 'function') {
```

Comment:
Should this be removed?  I'm concerned that someone may set `file._contents` to a non-wrapped stream.